v19.1/remove-nodes.md

```diff
@@ -394,6 +456,11 @@ $ cockroach node status --decommission --insecure --host=<address of any live no
 (5 rows)
 ~~~
 
+- `is_decommissioning` will be `true` for a node that is undergoing or has completed the decommissioning process. This indicates that replicas are being or have been transferred to other nodes.
+- `is_draining` will be `true` for a node that has been stopped. This indicates that SQL connections are no longer being accepted by the node.
```

Comment:
Nodes may be stopped and `is_draining == false` as well. Just in case this may cause confusion.